clients/drcachesim/common/trace_entry.h

```diff
@@ -472,6 +476,16 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_MAYBE_BLOCKING_SYSCALL,
 
+    /**
+     * Indicates a point in the trace where a syscall's trace start.
+     */
+    TRACE_MARKER_TYPE_SYSCALL_TRACE_START,
```

Comment:
Add printing code for every new marker to tools/view.cpp.

Reply:
Done.